bin/lib/zopen-build

```diff
@@ -113,7 +113,7 @@ export utilparentdir="$(cd "$(dirname "$0")/../.." >/dev/null 2>&1 && pwd -P)"
 for tmp in "$TMPDIR" "$TMP" /tmp
 do
   if [ ! -z $tmp ] && [ -d $tmp ]; then
-    TEMP_C_FILE="$tmp/$LOGNAME.c"
+    export ZOPEN_TEMP_C_FILE="$tmp/$LOGNAME.c"
```

Comment:
need a check after this loop that you were able to set ZOPEN_TEMP_C_FILE - otherwise the rm -rf is pretty nasty below...